src/graphql-server/dnaInterfaces/hhaDnaInterface.js

```diff
@@ -1,25 +1,29 @@
-import { instanceCreateZomeCall, installHapp } from '../holochainClient'
+import { instanceCreateZomeCall } from '../holochainClient'
 
 export const INSTANCE_ID = 'hha' // holo-hosting-app
 const createZomeCall = instanceCreateZomeCall(INSTANCE_ID)
 
-export const HhaDnaInterface = {
+const HhaDnaInterface = {
   currentUser: {
-    create: (host_doc) => createZomeCall('host/register_as_host')({host_doc}),
-    get: () => createZomeCall('host/is_registered_as_host')(),
+    // create is getting passed a variable (hostDoc) but is currently ignoring it
+    create: async () => {
```

Comment:
here and in `get` below, I'm transforming the data coming back from the zome so that it is the shape graphQL types expect. in this case, a javascript object with a single `id` field.

the `links` stuff is how the `host/is_registered_as_host` zome call returns its data